server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -89,6 +90,7 @@ export class DeliLambdaFactory extends EventEmitter implements IPartitionLambdaF
         } catch (error) {
             const errMsg = "Deli lambda creation failed";
             context.log?.error(`${errMsg}. Exception: ${inspect(error)}`, { messageMetaData });
+            Lumberjack.error(`${errMsg}. Exception: ${inspect(error)}`, { messageMetaData });
```

Comment:
I think we do not need to add an extra set of curly braces around `messageMetaData`. It was probably used for the context logger for legacy reasons. But in Lumberjack, it only cares about a `properties` object. `messageMetaData` is already an object that includes two properties: `documentId` and `tenantId`. We want those being top level properties to make sure our telemetry pipeline is capable of populating the correct fields in our log tables. I believe that would not happen if you do `{ messageMetaData }`, since in that case, you would have a new object where the top level property is called `messageMetaData` and the value is another object `{ "documentId": "abc", "tenantId": "123" }`.

To make things even better, and to stay consistent with the rest of the code, let's not use `messageMetaData` for Lumberjack. instead, can you update this and all the other occurrences to use getLumberBaseProperties? And as per my other comment, let's also extract the error into the error property.

here, you could do:
```ts
Lumberjack.error(errMsg, getLumberBaseProperties(documentId, tenantId), error);
```